<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>820</width>
    <height>435</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="margin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>3</number>
    </property>
    <item row="0" column="0">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="page_1">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QPushButton" name="pushButtonAdd">
          <property name="toolTip">
           <string>Нажмите, чтобы добавить нового клиента</string>
          </property>
          <property name="text">
           <string>Добавить</string>
          </property>
          <property name="icon">
           <iconset resource="Resources.qrc">
            <normaloff>:/new/prefix1/images/Add.png</normaloff>:/new/prefix1/images/Add.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
          <property name="shortcut">
           <string>Ctrl+N</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="pushButtonEdit">
          <property name="toolTip">
           <string>Нажмите, чтобы отредактировать данные уже созданного клиента</string>
          </property>
          <property name="text">
           <string>Изменить</string>
          </property>
          <property name="icon">
           <iconset resource="Resources.qrc">
            <normaloff>:/new/prefix1/images/Edit.png</normaloff>:/new/prefix1/images/Edit.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
          <property name="shortcut">
           <string>Ctrl+E</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="6">
         <widget class="QTableView" name="tableView"/>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="pushButtonFind">
          <property name="toolTip">
           <string>Нажмите, чтобы ввести поисковый запрос по базе</string>
          </property>
          <property name="text">
           <string>Найти</string>
          </property>
          <property name="icon">
           <iconset resource="Resources.qrc">
            <normaloff>:/new/prefix1/images/Find.png</normaloff>:/new/prefix1/images/Find.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
          <property name="shortcut">
           <string>Ctrl+F</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QPushButton" name="pushButtonRemove">
          <property name="toolTip">
           <string>Нажмите, чтобы удалить запись о клиенте</string>
          </property>
          <property name="text">
           <string>Удалить</string>
          </property>
          <property name="icon">
           <iconset resource="Resources.qrc">
            <normaloff>:/new/prefix1/images/Remove.png</normaloff>:/new/prefix1/images/Remove.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
          <property name="shortcut">
           <string>Del</string>
          </property>
         </widget>
        </item>
        <item row="0" column="5">
         <widget class="QPushButton" name="pushButtonManual">
          <property name="text">
           <string>Инструкция</string>
          </property>
          <property name="icon">
           <iconset resource="Resources.qrc">
            <normaloff>:/new/prefix1/images/Manual.png</normaloff>:/new/prefix1/images/Manual.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
          <property name="shortcut">
           <string>F1</string>
          </property>
         </widget>
        </item>
        <item row="0" column="4">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_2">
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Добавление клиента №</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <layout class="QGridLayout" name="gridLayout_3">
              <item row="0" column="0">
               <widget class="QLabel" name="labelLastName">
                <property name="text">
                 <string>Фамилия</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="labelFirstName">
                <property name="text">
                 <string>Имя</string>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="labelMiddleName">
                <property name="text">
                 <string>Отчество</string>
                </property>
               </widget>
              </item>
              <item row="3" column="0">
               <widget class="QLabel" name="labelBirthDay">
                <property name="text">
                 <string>Дата рождения</string>
                </property>
               </widget>
              </item>
              <item row="4" column="0">
               <widget class="QLabel" name="labelPhoneNumber">
                <property name="text">
                 <string>Телефонный номер</string>
                </property>
               </widget>
              </item>
              <item row="5" column="0">
               <widget class="QLabel" name="labelEMail">
                <property name="text">
                 <string>E-Mail</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout">
              <item>
               <widget class="QLineEdit" name="lineEditLastName">
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEditFirstName">
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEditMiddleName">
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDateEdit" name="dateEditBirthDay">
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
                <property name="displayFormat">
                 <string>dd.MM.yyyy</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEditPhoneNumber">
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEditEMail">
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <item>
               <widget class="QLabel" name="labelCarBrand">
                <property name="text">
                 <string>Марка автомобиля</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="labelCarModel">
                <property name="text">
                 <string>Модель автомобиля</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="labelCarYear">
                <property name="text">
                 <string>Год выпуска</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="labelCarColor">
                <property name="text">
                 <string>Цвет</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="labelCarVIN">
                <property name="text">
                 <string>VIN номер</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="labelCarNumber">
                <property name="text">
                 <string>Гос. номер</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <item>
               <widget class="QLineEdit" name="lineEditCarBrand">
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEditCarModel">
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDateEdit" name="dateEditCarYear">
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
                <property name="currentSection">
                 <enum>QDateTimeEdit::YearSection</enum>
                </property>
                <property name="displayFormat">
                 <string>yyyy</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="comboBoxCarColor">
                <property name="editable">
                 <bool>true</bool>
                </property>
                <item>
                 <property name="text">
                  <string>Серебряный</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Черный</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Красный</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Зеленый</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Синий</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Белый</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Серый</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Бежевый</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Коричневый</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Черный</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Золотой</string>
                 </property>
                </item>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEditCarVIN">
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEditCarNumber">
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>558</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="pushButtonSave">
              <property name="text">
               <string>Сохранить</string>
              </property>
              <property name="shortcut">
               <string>Return</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButtonCancel">
              <property name="text">
               <string>Отмена</string>
              </property>
              <property name="shortcut">
               <string>Esc</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page">
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="0">
         <widget class="QTextBrowser" name="textBrowser">
          <property name="html">
           <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;База данных Smolsto&lt;/span&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Инструкция&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;База данных предназначена для удобного хранения данных о клиентах компании и быстрого их редактирования.&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:11pt;&quot;&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:11pt; font-weight:600;&quot;&gt;Добавление клиента в БД:&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;- Для добавления нового клиента в базу данных необходимо нажать на кнопку &amp;quot;Добавить&amp;quot; на главном окне программы или комбинацию клавиш &amp;quot;Ctrl+N&amp;quot;.&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;- В открывшейся форме нужно указать все необходимые данные, при этом все незаполненные данные автоматически станут &amp;quot;-&amp;quot;.&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;- После того, как все данные будут добавлены, нажимаем &amp;quot;Сохранить&amp;quot;, либо &amp;quot;Отмена&amp;quot; для отмены добавления. Также завершить или отменить добавление можно клавишами &amp;quot;Enter&amp;quot; и &amp;quot;Esc&amp;quot; соответственно.&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:11pt;&quot;&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:11pt; font-weight:600;&quot;&gt;Редактирование данных клиента:&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;- Для того, чтобы отредактировать данные, нужно выделить любое количество элементов строки, которую хотите изменить. Если выделить лишь одну ячейку, то курсор автоматически переместится в соответствующую ячейку для быстрого редактирования.&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;- После выделения необходимо нажать кнопку &amp;quot;Редактировать&amp;quot; на главном окне программы или комбинацию клавиш &amp;quot;Ctrl+E&amp;quot;.&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;- После того, как все данные будут отредактированы, нажимаем &amp;quot;Изменить&amp;quot;, либо &amp;quot;Отмена&amp;quot; для отмены редактирования. Также завершить или отменить редактирование можно клавишами &amp;quot;Enter&amp;quot; и &amp;quot;Esc&amp;quot; соответственно.&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:11pt;&quot;&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:11pt; font-weight:600;&quot;&gt;Поиск по БД:&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;- Для поиска по базе данных необходимо нажать кнопку &amp;quot;Найти&amp;quot; на главном окне программы или комбинацию клавиш &amp;quot;Ctrl+F&amp;quot;.&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;- В появившемся диалоговом окне выбраем столбкик (т.е. параметр), по которому будет вестись поиск и набираем фразу для поиска.&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;- После ввода всех необходимых данных нажимаем кнопку &amp;quot;Найти&amp;quot; или клавишу &amp;quot;Enter&amp;quot;, или кнопку &amp;quot;Отмена&amp;quot; или &amp;quot;Esc&amp;quot; для отмены поиска.&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;- Для того, что вновь просмотреть базу данных полностью, нужно сделать поиск по пустой строке (т.е. нажимаем &amp;quot;Найти&amp;quot; на главном окне программы и, ничего не меняя в открывшемся окне, нажимаем &amp;quot;Найти&amp;quot;)&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:11pt;&quot;&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:11pt; font-weight:600;&quot;&gt;Удаление клиентов:&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;- Для того, чтобы удалить данные, нужно выделить любые желаемые строчки (можно одну, можно несколько подряд, можно любые, зажав клавишу Ctrl), нажать кнопку &amp;quot;Удалить&amp;quot; на главном окне программы или клавишу &amp;quot;Del&amp;quot; и подтвердить удаление.&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:11pt;&quot;&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:11pt; font-weight:600;&quot;&gt;Инструкция:&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;- Просмотреть инструкцию можно по нажатию кнопки &amp;quot;Инструкция&amp;quot; или клавиши &amp;quot;F1&amp;quot; &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="pushButtonToMain">
          <property name="text">
           <string>Вернуться на главную страницу программы</string>
          </property>
          <property name="shortcut">
           <string>Esc, Return</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>820</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>lineEditLastName</tabstop>
  <tabstop>lineEditFirstName</tabstop>
  <tabstop>lineEditMiddleName</tabstop>
  <tabstop>dateEditBirthDay</tabstop>
  <tabstop>lineEditPhoneNumber</tabstop>
  <tabstop>lineEditEMail</tabstop>
  <tabstop>lineEditCarBrand</tabstop>
  <tabstop>lineEditCarModel</tabstop>
  <tabstop>dateEditCarYear</tabstop>
  <tabstop>comboBoxCarColor</tabstop>
  <tabstop>lineEditCarVIN</tabstop>
  <tabstop>lineEditCarNumber</tabstop>
  <tabstop>pushButtonSave</tabstop>
  <tabstop>pushButtonCancel</tabstop>
  <tabstop>pushButtonRemove</tabstop>
  <tabstop>tableView</tabstop>
  <tabstop>pushButtonFind</tabstop>
  <tabstop>pushButtonEdit</tabstop>
  <tabstop>pushButtonAdd</tabstop>
 </tabstops>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections/>
</ui>
